server/sockets.js

```diff
@@ -143,6 +143,9 @@ async function saveHistory(boardName, message) {
 		case "delete":
 			if (id) board.delete(id);
 			break;
+		case "deleteall":
+			board.deleteall();
```

Comment:
We should check that the tool is activated, and this tool should probably not be activated by default.